### ISSUE TITLE
Doesnt compile

### ISSUE THREAD

**Issue by thetechknight** [andlabs/tms34010disasm#1]
This thing isnt compiling. Do you have a binary?

There are issues like missing function Disassemble, I dont see the one Go file imported into the other one, Also there is a Paranthesis missing on one of the lines, something about Bytes being undefined. List goes on. 

help?

**Comment by andlabs** [comment]
No, I'm sorry. I never made much progress with this; the code here is practically nonexistent.

**Comment by thetechknight** [comment]
Darn. I was hoping something worked. even the MAME version didnt work. I was able to get someone to fix it though. It seems as if nobody supports this CPU as far as disassembly, not even IDA. I guess it must have been one of those flops back then?

**Comment by andlabs** [comment]
I don't know if this chip was a flop; it's just that it's a rather unusual CPU. As for this codebase, I started it because IDA didn't have it, and it was forked from another already-in-progress disassembler that I wrote; I just never got around to finishing it.

**Comment by thetechknight** [comment]
Ah ok. Yea I am noting the rather strange architecture of this thing. like for example the lower 4 address bits arnt used and it means it can only increment addressing every 16 bytes, thats the part that just boggles my mind.

**Comment by andlabs** [comment]
Oh no, it's *bit*-addressed, not *byte*-addressed.

**Comment by thetechknight** [comment]
Yea I hear that. but I am having a hard time wrapping my head around that, logically.

**Comment by thetechknight** [comment]
So if reading XXXXXXD0 would read the entire 16 bit word? or would it only read the first bit? D1 second bit? is that how it works?

**Comment by andlabs** [comment]
I wouldn't know, sorry.
